### ISSUE TITLE
Gesso Accessibility Audit (Storybook)

### ISSUE THREAD

**Issue by jeanette-stair-forumone** [forumone/gesso#912]
https://forumone.github.io/gesso/?path=/story/pages-article--article

**Comment by jeanette-stair-forumone** [comment]
@kmonahan or @dcmouyard can one of you assign this to me (make it so I can be assigned to issues)?

**Comment by kmonahan** [comment]
@jeanette-stair-forumone Assigned the issue to you and added you to the project (should be an invite in whatever email inbox you use with GitHub). Let me know if we need to bump up your permissions level.

**Comment by jeanette-stair-forumone** [comment]
Did a review of pages-page--page
[Assessment_20250102_pagespagepage.json](https://github.com/user-attachments/files/18292160/Assessment_20250102_pagespagepage.json)

## Issues
- Keyboard - Main nav - cannot expand About and Resources when using screen reader (NVDA). Expected: https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/ Enter or Space should activate the disclosure control and toggle the visibility of the disclosure content. When not using a screen reader, space or up/down arrows toggles the control but enter doesn't (and should)
- Headings - same issue as discussed in https://github.com/forumone/gesso/issues/911 
- Links - need to discuss strategy for the site logo link and "Home" in the breadcrumb. These go the same place (the homepage) but have different accessible names ("Site Name" and "Home"). Accessibility Insights for Web says "Links with the same destination should have the same link text." but I don't believe this is a real failure, though it might not be best practice. I'd like to discuss with the accessibility team at Forum One and ask the IAAP forum for opinions.
- Adaptable content 
  - Heading level 6 does not respond to change in styling 
  - Hover content in main menu is not dismissible - expect to be able to use ESC to close it without moving pointer hover or keyboard focus
- Some code is flagged because it includes "position: absolute;" Need a developer to review if this is necessary and if it's likely to cause downstream issues. The requirement is that any positioned content retains its meaning when the page is linearized.
- Contrast - Back to top icon does not have enough contrast (should have 3:1) [Fixed by #950 ]

**Comment by jeanette-stair-forumone** [comment]
@kmonahan I'm not sure the process here, but reassigning to you. Let me know what other information would help

**Comment by kmonahan** [comment]
@jeanette-stair-forumone There's not really a process. People pick up work as they have time. I'm mostly focused on Gesso for React currently so unassigning from myself.
